controllers/weather.controller.js

```diff
@@ -0,0 +1,20 @@
+import axios from "axios";
+
+import { requestOptions } from "../utils/requestOptions.js";
+
+export class WeatherController {
+  static async getWeather(req, res) {
+    await axios
+      .request(requestOptions(req.query.cityName))
```

Comment:
better use requestOptions(req.query)

Reply:
Better to remove utils and add service layer that will receive just a city name and will encapsulate building of the request object and working with axios. Controller layer should be thin, it is just a proxy between UI and application logic. Otherwise If you decide to create CLI interface for your app you will have too implement the same logic in CLI commands